crates/containerd-shim-wasm/src/sandbox/cgroups/mod.rs

```diff
@@ -286,13 +287,35 @@ mod tests {
         .map_err(|e| Error::Others(format!("failed to apply cgroup: {}", e)))
     }
 
+    struct CgWrapper {
+        cg: Box<dyn Cgroup>,
+    }
+
+    impl CgWrapper {
+        fn new(p: &str) -> Self {
+            Self {
+                cg: p.try_into().unwrap(),
+            }
+        }
+    }
+
+    impl Drop for CgWrapper {
+        fn drop(&mut self) {
+            self.cg.delete().unwrap();
+        }
+    }
+
     #[test]
     fn test_cgroup() -> Result<()> {
         if !super::super::exec::has_cap_sys_admin() {
             println!("running test with sudo: {}", function!());
             return run_test_with_sudo(function!());
         }
+
         let cg = new("containerd-wasm-shim-test_cgroup".to_string())?;
+
+        println!("cgroup: {}", cg.version());
```

Comment:
Is this here for debugging?

Reply:
Yes, it's useful if this fails later to know what version of groups the test ran against.

Reply:
What do you think about `log::debug!()` rather than `println!()`?

Reply:
Changed.